OpenRA.Mods.Common/ActorExts.cs

```diff
@@ -141,6 +141,10 @@ public static void NotifyBlocker(this Actor self, IEnumerable<CPos> positions)
 					return false;
 			}
 
+			// Occupied by any actor, including ally/friendly units
+			if (self.World.ActorMap.GetActorsAt(pos).Any())
```

Comment:
Another thing is, in RA2 they have terrorists and it makes sense not to crush them! BTW, thinking about this, with the above code, I think the harvesters will avoid enemy anti vehicle mines automatically, oopsy